### PR TITLE
Fix Content Creation Error

### PR DESCRIPTION
**Pull request by chouseknecht**
Proposed fix for #775 and #776

If merged, will need to be cherry-picked to 3.0.1

### CODE REVIEW

File: galaxy/worker/importers/base.py [hunk 1]
```diff
@@ -65,17 +66,26 @@ def make_content(self):
...
-            defaults={
-                'name': self.translate_content_name(name),
-                'is_valid': False,
-            }
-        )
-
+        try:
+            # Check for an existing Content object matching name
+            is_created = False
+            obj = models.Content.objects.get(namespace=ns,
```
Comment (cutwater) [inline galaxy/worker/importers/base.py]: Please try to fit your code to 80 characters line width.

File: galaxy/worker/importers/base.py [hunk 2]
```diff
@@ -101,7 +111,7 @@ def _update_readme(self, content):
 
     def _log_create_content(self, content_id, is_created):
         action = 'Created new' if is_created else 'Found'
-        self.log.debug(
+        LOG.debug(
```
Comment (cutwater) [inline galaxy/worker/importers/base.py]: Why this change?

Comment (chouseknecht) [reply galaxy/worker/importers/base.py]: Bad. Will revert that. It's what I had to do in order to see the output. The self.log.debug() messages don't surface in the API response for some reason, and thus don't appear on the UI.

File: galaxy/worker/importers/base.py (hunk 1 above)
Comment (chouseknecht) [reply galaxy/worker/importers/base.py]: OK.
